client/core/trade.go

```diff
@@ -1386,10 +1386,13 @@ func (t *trackedTrade) checkSwapFeeConfirms(match *matchTracker) bool {
 		// Confirmed will be set in the db.
 		return true
 	}
+	// Waiting until the swap is definitely confirmed in order to not
+	// keep calling the fee checker before the swap is confirmed.
+	mySwapConfs, _ := match.confirms()
 	if match.Side == order.Maker {
-		return match.Status >= order.MakerSwapCast
+		return match.Status > order.MakerSwapCast || mySwapConfs > 0
```

Comment:
Why `>` instead of `>=`? Why should maker wait to see the taker's swap before checking for fees on its own swap?

Reply:
> We are not strictly tracking the confirmations on the swap transaction, and due to the irregular way the confirmations were coming in when testing on testnet, I thought this would be safer.

Hmm, that's an interesting point.  I suppose there can be a case where no `tick` gets your own swap's confirms before moving on to the next status, which could preclude the confs check.

OK, I think this is safest since tick does not make explicit guarantees about this.

Actually, I think the counter party could even make their transaction without waiting for a single confirmation on ours.  Nothing stopping that.